### DISCUSSION TITLE
Consistent tag and architecture platform

### DISCUSSION THREAD

**Discussion by timdinhdotcom** [aws/aws-lambda-dotnet#1364] [General]
I noticed that the tag dotnet:6 is an arm64 while dotnet:7 is an amd64. This could potentially causing issues for a lot of people. I proposed that we are consistent with the platform architecture. Therefore, dotnet:7 should be arm64 instead of amd64

**Comment by ashishdhingra** [comment]
@timdinhdotcom Please refer https://github.com/aws/aws-lambda-dotnet/issues/1471#issuecomment-1488908142. This is fixed.

@github-actions proposed-answer
